src/sql/src/rbac.rs

```diff
@@ -259,23 +188,28 @@ pub fn check_plan(
 ) -> Result<(), UnauthorizedError> {
     rbac_preamble!(catalog, role_metadata, session_vars);
 
-    check_item_usage(
+    // Obtain all roles that the current session is a member of.
+    let role_membership = catalog.collect_role_membership(&role_metadata.current_role);
+
+    let rbac_requirements = generate_rbac_requirements(
```

Comment:
Given the [latest guidance on logging](https://www.notion.so/materialize/Logs-Logging-Guide-6e6c4d0d0bd44cdea8773157bbc3e317#07f289dcd002458c9426b99b3e71364b) could you add a `tracing::debug!` here that logs the `RbacRequirements`? Could be helpful in the future if we need to figure out why a user doesn't have permissions for something

Reply:
Done